packages/fast-components-react-base/src/toggle/toggle.spec.tsx

```diff
@@ -39,6 +39,12 @@ describe("toggle", (): void => {
         expect((Toggle as any).name).toBe(Toggle.displayName);
     });
 
+    test("should have correct input type checkbox", () => {
```

Comment:
Should we test for the root in an additional test block as well as the nested element and type?

Reply:
Again, I would say it is unnecessary.